.env.sample

```diff
@@ -1,2 +1,5 @@
 CRATESFYI_GITHUB_USERNAME=
 CRATESFYI_GITHUB_ACCESSTOKEN=
+CRATESFYI_PREFIX=https://localhost:3000
```

Comment:
`CRATESFYI_PREFIX` should be a file path, not a url, probably `ignored/cratesfyi-prefix` in the sample?